java/java-analysis-impl/src/com/intellij/codeInspection/dataFlow/java/inliner/StreamChainInliner.java

```diff
@@ -69,6 +69,7 @@
     instanceCall(JAVA_UTIL_STREAM_STREAM, "reduce").parameterTypes("T", "java.util.function.BinaryOperator");
   private static final CallMatcher COLLECT_TERMINAL =
     instanceCall(JAVA_UTIL_STREAM_STREAM, "collect").parameterTypes("java.util.stream.Collector");
+  private static final CallMatcher COLLECT_TO_LIST_TERMINAL = instanceCall(JAVA_UTIL_STREAM_STREAM, "toList");
```

Comment:
Please rename it to `TO_LIST_TERMINAL`. Naming here assumes mentioning of called methods but you are not calling `collect()` here.

Reply:
Renamed